job-server-tests/src/main/java/spark.jobserver/JavaHelloWorldJob.java

```diff
@@ -1,12 +1,12 @@
-package spark.jobserver;
+// package spark.jobserver;
```

Comment:
@velvia @zeitos It should be something like this, but the jobserver itself needs to account for the different class. The ```type C``` stuff as well as the ```case object``` do not translate well to java, so they don't compile. But also take note of the path I changed as also Zeitos suggested...

https://github.com/hntd187/spark-jobserver/commit/825d02e7051dd73ed5c2acf77a6836e6efb9950a